wikichange/dist/content.js

```diff
@@ -0,0 +1,334 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
I am assuming this file is generated by npm to feed into chrome?

Reply:
Yes. Check the README.md, the dist folder is the output you want to load into chrome